core/services/relay/evm/types/types.go

```diff
@@ -207,8 +207,10 @@ type RelayConfig struct {
 	SendingKeys pq.StringArray `json:"sendingKeys"`
 
 	// Mercury-specific
-	FeedID                  *common.Hash `json:"feedID"`
-	EnableTriggerCapability bool         `json:"enableTriggerCapability"`
+	FeedID                   *common.Hash `json:"feedID"`
+	EnableTriggerCapability  bool         `json:"enableTriggerCapability"`
+	TriggerCapabilityName    string       `json:"triggerCapabilityName"`
```

Comment:
how is the plumbing going to work?

i have wf spec that declares i want to use v1.2.3.

i have to copy and paste v1.2.3 to some other config? specifically the we'll need a job per capability version on the asset don?

 i don't get it, please help me understand

Reply:
Nodes in Asset DON configure which version of the capability they support. For example in staging the current DON defaults to 1.0.0 but the new DON will set it to 1.1.0 (because it only server feed IDs in the new format). If we kept both DONs then a workflow spec could choose the version (in practice we will deprecate the old DON soon).

Reply:
and how do the asset nodes config it is my question?

by a) updating contract b) running a spec job that has keyed relation with the values in the contract

so there is no unified view of the truth b/c the job could be misconfigured wrt the contract.

is this correct?

Reply:
Yes, local capabilities always need to match what's in the Registry. That's especially true with standard capabilities - if somebody adds a local spec with a version not matching the onchain registry, it won't work.